packages/react-components/src/components/Search/Search.tsx

```diff
@@ -128,11 +133,16 @@ export const SearchInput: React.FC<ISearchInputProps> = ({
         disabled={isDisabled}
         kind="primary"
       />
-      <Text as="div">
+      <Text
+        as="div"
+        className={cx(
+          styles[inputBaseClass],
```

Comment:
I think we need to rename this class, because it no longer addresses the input element